kubernetes/samples/scripts/create-weblogic-domain/manually-create-domain/domain.yaml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# This is an example of how to define a Domain resource.  Please read through the comments which explain
+# what updates are needed.
+#
+apiVersion: "weblogic.oracle/v2"
+kind: Domain
+metadata:
+  # Update this with the `domainUID` of your domain:
+  name: domain1
+  # Update this with the namespace your domain will run in:
+  namespace: weblogic
```

Comment:
'weblogic' probably isn't a good default namespace name - how about 'default' ?

Reply:
+1

Reply:
+1

Reply:
Yes, and 'default' is the default in the helm chart, correct?